### PR TITLE
[dagster-airlift][rfc] readme with proxying language

### PR DESCRIPTION
**Pull request by dpeng817**
Put together a version of the readme which uses "proxying" in place of "migrating". Tried to be thoughtful with language replacement.

How do we feel about this?

### CODE REVIEW

**Comment by dpeng817** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24610?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#24610** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24610?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#24569** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24569?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24556** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24556?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24540** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24540?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @dpeng817 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/experimental/dagster-airlift/examples/tutorial-example/README.md [hunk 1]
```diff
@@ -270,76 +270,77 @@ Kicking off a run of the DAG in Airflow, you should see the newly created assets
 
 _Note: There will be some delay between task completion and assets materializing in Dagster, managed by the sensor. This sensor runs every 30 seconds by default (you can reduce down to one second via the `minimum_interval_seconds` argument to `sensor`), so there will be some delay._
 
-## Migrating Assets
+## Proxying to Assets
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/tutorial-example/README.md:273]: So to be clear I still think we should describe this as "Migrating to Assets". Proxying is our _tool_ for facilitating the process.

File: examples/experimental/dagster-airlift/examples/tutorial-example/README.md [hunk 2]
```diff
@@ -270,76 +270,77 @@ Kicking off a run of the DAG in Airflow, you should see the newly created assets
...
 tasks:
   - id: load_raw_customers
-    migrated: False
+    proxied: False
   - id: build_dbt_models
-    migrated: False
+    proxied: False
   - id: export_customers
-    migrated: False
+    proxied: False
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/tutorial-example/README.md:290]: I think this is great and makes it much more clear what is going on. This is tracking a specific technical behavior, not arbitrary metadata for bookkeeping.

**Review by schrockn — CHANGES_REQUESTED** [review]
Inline comments

**Pull request closed without merging**